### PR TITLE
update release info in FAQ for 2019 Aug and Nov releases

### PR DESCRIPTION
**Pull request by yhu6**
Signed-off-by: Yong Hu <yong.hu@intel.com>

### CODE REVIEW

File: site/faq/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The StarlingX team is actively working with a number of other projects and commu
 
 #### When is the first release of StarlingX?
 
-The first release will be in October 2018. The project will release on a 4 month cadence each March, July and November in 2019. Releases are time based.
+The first release was in October 2018, the second release was available in August 2019, and approximately the third release would be in November 2019. After these 3 releases, the project will follow a 6-month release cadence, or in another word, a time based release model, similar to other Open Source projects, such as OpenStack.
```
Comment (ildikov) [inline site/faq/index.md]: I wonder if we should update the planned release cadence once the project had at least one successful 6-month cycle?

Comment (yhu6) [reply site/faq/index.md]: I talked to Ghada and she is going to share me this release related information. After that, I will update this PR.

Comment (jimmytipit) [reply site/faq/index.md]: I'm going to push this change through with my other pull request since the final language was dictated there.

**Comment by jimmytipit** [comment]
I'm going to close this since it's addressed here: https://github.com/StarlingXWeb/starlingx-website/pull/42

**Pull request closed without merging**